service-info/README.md

```diff
@@ -0,0 +1,55 @@
+# GA4GH Service Info Types
+
+This file details how to add values to the list of Service Info type values for the GA4GH API.
+
+## Background
+
+To enable servers using the suite of GA4GH APIs to be used in a networks, the Service Info endpoint was created. A centrally maintained list of the type values for GA4GH APIs was requested, which was to be managed by the GA4GH TASC Force. See [here for the main Service Info repository](https://github.com/ga4gh-discovery/ga4gh-service-info).
+
+## Using These Files
+
+There are two accompanying files here which should have the same information in both formats.
+
+### JSON format
+
+The JSON file shall features entries in the following format:
+```
+[
+  {
+    "type": {
+      "group": "org.ga4gh",
+      "artifact": "first-api-name",
+      "version": "1.0.0"
```

Comment:
What does `version` indicate? The first version of the artefact that included service-info? The current version of the artefact? (If the latter, what is the procedure by which TASC should keep this up to date?)

Should the JSON file include a link to the specification for each artefact?

Reply:
Having version does go above and beyond the original steering committee request for the group and artifact items being listed centrally. It might be onerous to list all versions. IIRC there was a thought to link these to the specification itself, which might then make this useful.

Reply:
Following up on the points raised on the call today, +1 for removing. It seems like unnecessary overhead to maintain the current version in 2 places, and since everybody is using semver, it's pretty well-defined which versions get approved.

Reply:
version has now been removed